server/storage/schema/confstate_test.go

```diff
@@ -12,14 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package buckets
+package schema
```

Comment:
Since this is a public package and not internal would this be considered breaking change? I know some projects import etcd for example.

Reply:
Is there any official policy what package have stable functions/interface? It makes sense for libraries, but it's strange for me that we would have something like that for server code. Do we care if someone imports a package that we haven't officially commit to stabilize?

 I mean that in a way that maybe we should start an effort to decide what packages are stable and which are definitely not.

Reverted this change.

Reply:
Agreed, I don't think there is an official policy, but its not part of an `internal` directory. But might be nice to discuss this in our next meeting, what do you think?

Reply:
SG